src/one_dragon/devtools/python_launcher.py

```
@@ -306,10 +319,12 @@ def run_python(app_path, no_windows: bool = True, args: list[str] | None = None,
         ctx = OneDragonEnvContext()
         configure_environment(ctx, cwd)
         fetch_latest_code(ctx)
+        sync_dependencies(ctx)
         execute_python_script(ctx, app_path, no_windows, args, piped)
     except SystemExit as e:
         print_message(f"程序已退出，状态码：{e.code}", "ERROR")
+        raise
     except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)